#!/usr/bin/env python

PACKAGE = "autonomous_bicycle"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("torque_wheel", double_t, 0, "torque (N/m)", -1, -1, 1000)
gen.add("vel_steering", double_t, 0, "velocity (rad/s)", 0, -5, 5)

exit(gen.generate(PACKAGE, "autonomous_bicycle", "bicycle_interaction"))